src/dev-journal.js

```diff
@@ -84,7 +84,7 @@ const getLWDJournals = async (DEV_JOURNAL, { backFromVacay, isOnVacay }) => {
   }))
 
   // add in prev-vacay tasks as prev-workday tasks
-  prevDayTasks = [...prevDayTasks, ...prevVacayTasks]
+  prevDayTasks = [...prevDayTasks.filter((r) => r), ...prevVacayTasks.filter((r) => r)]
```

Comment:
conceptually simpler:

```suggestion
  prevDayTasks = [...prevDayTasks, ...prevVacayTasks].filter((r) => r)
```